src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -533,11 +557,15 @@ private void CheckClientTimeout()
             }
         }
 
-        private static void AbortConnection(object state)
+        private static async void AbortConnection(object state)
```

Comment:
Yes I know... open to suggestions

Reply:
Async void.....

Reply:
More "async avoid", amirite?

I'll show myself out 🤦